geomesa-cassandra/geomesa-cassandra-datastore/src/main/scala/org/locationtech/geomesa/cassandra/data/CassandraFeatureStore.scala

```diff
@@ -65,6 +66,12 @@ class CassandraFeatureStore(entry: ContentEntry) extends ContentFeatureStore(ent
   }
 
 
+  override def getInfo: ResourceInfo = {
```

Comment:
I would say it should go in a new trait here: https://github.com/locationtech/geomesa/tree/master/geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools

but @elahrvivaz would have a better idea.